Gemfile

```diff
@@ -9,14 +9,12 @@ end
 
 gemspec
 
-gem 'actionmailer', '< 6.1'
-gem 'activesupport', '< 6.1'
 gem 'appraisal', '~> 2.2'
 gem 'honeybadger', '~> 4.11'
 gem 'pry'
 gem 'pry-byebug'
 gem 'rake'
-gem 'rspec'
+gem 'rspec', '~> 3.10.0'
```

Comment:
This seems really narrow, do we need to limit it?

Reply:
Yes, there are some bugs for the versions above, both 3.11.x and 3.12.x.